src/UI/Implementation/Component/Button/Renderer.php

```diff
@@ -66,16 +66,22 @@ protected function renderButton(Component\Button\Button $component, RendererInte
 		if ($component->isActive()) {
 			// The actions might also be a list of signals, these will be appended by
 			// bindJavascript in maybeRenderId.
-			if (is_string($action)) {
+			if (is_string($action) && $action != "") {
```

Comment:
Why would the `$action` be empty here?

Reply:
If you have a custom JS button using withOnloadCode it seems to be impossible to prevent this action handling being fired (and redirect) otherwise.

Reply:
@tfamula

Reply:
I very much suspect this change to break some of the tests.